src/zipAll.js

```diff
@@ -0,0 +1,39 @@
+var _curry1 = require('./internal/_curry1');
+
+/**
+* Creates a new list out of the lists supplied by matching
+* equally positioned items from both lists. The returned list is
+* truncated to the length of the shorter of the two input lists.
+*
+* @func
+* @memberOf R
+* @category List
+* @sig [a] -> [a]
+* @param {Array} List of arrays to consider
+* @return {Array} The list made by paring up same indexed elements of supplied list.
+* @example
+*
+* R.zipAll([[1, 2, 3], ['a', 'b', 'c'], ['A', 'B', 'C']]) //=> [[1, 'a', 'A'], [2, 'b', 'B'], [3, 'c', 'C']]
```

Comment:
> I'd prefer not to mix types in the example.

I disagree.  I think that part of the point of this function is that it can be used to create arbitrary tuples.  I would actually prefer to have the third list contain something like `[true, true, false]`